packages/core/core/src/ParcelConfig.js

```diff
@@ -64,7 +64,7 @@ export default class ParcelConfig {
   namers: PureParcelConfigPipeline;
   runtimes: PureParcelConfigPipeline;
   packagers: GlobMap<ParcelPluginNode>;
-  validators: GlobMap<ExtendableParcelConfigPipeline>;
+  validators: $ReadOnlyArray<ParcelPluginNode>;
```

Comment:
nit:

```suggestion
  validators: PureParcelConfigPipeline;
```

Even though this isn't strictly a pipeline (plugins don't use the result from previous plugins), we seem to be using this for things like reporters so 🤷 

Alternatively, we could create another type alias that points to this that describes it more of a plugin "list".

Reply:
(Runtimes also don't use previous results, but they are applied in order so that might be another defining property of a pipline)

Speaking of that, doesn't removing the glob filters break the existing validators because they now run on every asset regardless of type?

Reply:
Existing examples of validators have their own glob rules (in eslintrc and tsconfig) to determine if they should run on particular file globs. Worth covering with a test to confirm the behavior when this happens though.